sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/nlp/automl_nlp_job.py

```diff
@@ -179,6 +246,15 @@ def featurization(self, value: Union[Dict, NlpFeaturizationSettings]) -> None:
             self.set_featurization(**value)
 
     def set_data(self, *, training_data: Input, target_column_name: str, validation_data: Input) -> None:
+        """Define data configuration for NLP job
```

Comment:
Could you add an inline example for this?

Reply:
This class is not to be instantiated by users